release/gcloud/Dockerfile

```diff
@@ -0,0 +1,34 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM golang:1.13-stretch AS build
+ENV CGO_ENABLED=0
+ENV GO111MODULE=on
+
+ARG KPT_VERSION=unknown
+
+WORKDIR /go/src/
+COPY go.mod .
+COPY go.sum .
+RUN go mod download
+COPY . .
+RUN go build -v -o /usr/local/bin/kpt -ldflags="-s -w -X github.com/GoogleContainerTools/kpt/run.version=$KPT_VERSION" ./
+
+FROM gcr.io/google.com/cloudsdktool/cloud-sdk:alpine
```

Comment:
Should probably use a pinned version instead. We don't want to have this change without our explicit consent.

Reply:
I've pinned this to `332.0.0-alpine` and updated our releasing steps to include updates to the base image versions prior to release.